lib/modules/manager/gradle/utils.ts

```diff
@@ -44,6 +53,11 @@ export function isDependencyString(input: string): boolean {
     return false;
   }
 
+  const mavenStyleRange = parseMavenStyleRange(tempVersionPart);
```

Comment:
I see no need for this here. Should be enough to have `versionLikeSubstring` match ranges, which it already can except for those containing `!` and spaces.

Reply:
Correct, it is more of a fuzzy search and no strict version input validation.